pkgs/tools/typesetting/tex/nix/default.nix

```diff
@@ -63,17 +64,20 @@ rec {
             { rootFile = baseNameOf (toString rootFile); src = key; }
             "${pkgs.perl}/bin/perl ${./find-includes.pl}");
 
+          searchPath' = [(dirOf key)] ++ searchPath;
```

Comment:
This makes `dirOf key` take precedence over `searchPath`, do we want that?

Reply:
I think there is no obviously right answer, however, what I was thinking is: if the file being imported is right next to a file (in the same folder), it seems like it should take precedence.

Reply:
I guess that should depend on what would be the use case for `searchPath`. Not obvious to me either.

Reply:
For my use case, it doesn't really matter either way.

What I am doing is: creating a separate derivation which calls `lhs2TeX` with the root `.tex` file as input. I want to be able to import other local files in my resulting root `.tex`. I don't want to analyze and copy the dependencies in the `lhs2TeX` derivation as it is duplicate work and wastes space.

Aside: I create a custom derivation because I think the current `lhs2TeX` functions from nixpkgs don't work, but that is a separate problem.